src/styles/card.scss

```diff
@@ -0,0 +1,95 @@
+.card {
+  font-family: Roboto, Arial, sans-serif;
+  width: 198px;
+  height: auto;
+  border: 1px solid $light-gray-color;
+  border-radius: 5px;
+  transition: 300ms;
+
+  &:hover {
+    scale: (1.2);
+    h1 {
+      color: $card-head-color-hover;
+    }
+  }
+
+  &__body {
+    padding: 32px 16px 16px;
+    width: 166px;
```

Comment:
```suggestion
    width: 100%;
```

Reply:
I can’t use this because it destroys everything inside the cards.
![Screenshot_7](https://github.com/mate-academy/layout_catalog/assets/110745125/37be549a-fe93-440d-b378-d449716e6143)

Reply:
You can use it but you have to define for * selector `box-sizing: border-box;`